clang/include/clang/Driver/Options.td

```diff
@@ -4977,7 +4977,7 @@ def analyzer_checker : Separate<["-"], "analyzer-checker">,
     #define GET_PACKAGES
     #define PACKAGE(FULLNAME)  FULLNAME ","
     #include "clang/StaticAnalyzer/Checkers/Checkers.inc"
-    #undef GET_PACKAGES
+#undef GET_PACKAGES
```

Comment:
Bad indentation?

Reply:
Accidental change, fixed in 0a351d1a1b556e14c1dd679bae13ea3e374631dc